cmd/minishift/cmd/start.go

```diff
@@ -188,6 +189,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	}
 
 	clusterUp(&config)
+	setMinishiftContext(&config, host)
```

Comment:
We do need to run this after cluster up because then only we have access to generated kube config file.